conda_build/post.py

```diff
@@ -320,7 +320,7 @@ def mk_relative(m, f):
     if not is_obj(path):
         return
 
-    if sys.platform.startswith('linux'):
+    if sys.platform.startswith('linux') or ('bsd' in sys.platform):
```

Comment:
While it is not really the issue of this exact pull-request, but why don't just use `platform.system()` (`import platform`) instead of this hacking around with `startswith`, `in`?